src/main/java/org/folio/rest/impl/util/EventConfigHelper.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.rest.impl.util;
+
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.folio.cql2pgjson.exception.CQL2PgJSONException;
+import org.folio.rest.persist.cql.CQLQueryValidationException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class EventConfigHelper {
```

Comment:
Can you add a private constructor?
https://sonarcloud.io/organizations/folio-org/rules?open=squid%3AS1118&rule_key=squid%3AS1118

Reply:
Added